lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -0,0 +1,280 @@
+            desc => "Outdir: " . $self->outdir .
+                " not found or not a directory",
+        );
+    }
+    return @errors;
+}
+
+#Get a somatic-variation build from input or from clinseq-build
+sub resolve_somvar {
+    my $self = shift;
+    my $somvar_build = $self->somvar_build;
+    unless($somvar_build) {
+        my $clinseq_build = $self->clinseq_build;
+        unless($clinseq_build) {
+            die $self->error_message("Please pass a somatic-variation
+                or clinseq build as input.");
+        }
+        $somvar_build = $self->get_best_somvar_build($clinseq_build);
+    }
+    return $somvar_build;
+}
+
+#Take Varscan SNVs and split into per-chromosome calls.
+sub split_snvs {
+    my $self = shift;
+    my $varscan_op = shift;
+    my $snv_prefix = shift;
+    my $awk_cmd;
+    #use subsample of chr1 variants for test
+    if($self->test) {
+        $awk_cmd = "awk '\$1==1 { if(NR\%1000 == 0) { print > \"$snv_prefix.\"\$1\".snp\" }}' $varscan_op";
+    } else {
+        $awk_cmd = "awk '{ print > \"$snv_prefix.\"\$1\".snp\" }' $varscan_op";
+    }
+    Genome::Sys->shellcmd(cmd => $awk_cmd);
+    return;
+}
+
+#Get Varscan SNVs from somatic-variation
+sub get_varscan_snvs {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $snv_prefix = shift;
+    my $dd = $somvar_build->data_directory;
+    my @varscan_snvs =
+    glob(File::Spec->join($dd, "variants/snv/varscan-somatic-*/snvs.hq.unfiltered"));
+    unless(scalar @varscan_snvs) {
+        die $self->error_message("Unable to find varscan SNVs in $dd");
+    }
+    $self->status_message("varscan snvs file found: " .
+        join("\t", @varscan_snvs));
+    $self->split_snvs($varscan_snvs[0], $snv_prefix);
+}
+
+#Get tumor/normal BAMs from somatic-variation
+sub get_tumor_normal_bam {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $tumor_bam = $somvar_build->tumor_model->last_succeeded_build->
+    whole_rmdup_bam_file;
+    my $normal_bam = $somvar_build->normal_model->last_succeeded_build->
+    whole_rmdup_bam_file;
+    unless (-e $tumor_bam and -e $normal_bam) {
+        die $self->error_message("tumor_bam $tumor_bam or normal_bam " .
+            "$normal_bam does not exist.");
+    }
+    return ($tumor_bam, $normal_bam);
+}
+
+#Get the reference-sequence build from the
+# ref-align builds used in somatic-variation
+sub reference_build {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $normal_refalign_build = $somvar_build->normal_build if ($somvar_build);
+    my $tumor_refalign_build = $somvar_build->tumor_build if ($somvar_build);
+    my @input_builds = ($normal_refalign_build,
+        $tumor_refalign_build, $somvar_build);
+    for my $build (@input_builds){
+        next unless $build;
+        my $m = $build->model;
+        if ($m->can("reference_sequence_build")){
+            return $m->reference_sequence_build;
+        }
+    }
+}
+
+#Filter out putative false-positive SNVs
+sub filter_snvs {
```

Comment:
This is a really good point since this filter takes up quite a bit of run-time. From what I can tell, the original filtering module "GMT::VarScan::SomaticParallelFilter.pm" is not being run as part of any build. There are probably other modules attempting similar filtering but not as stringent on the Gemline/LOH calls.
I might be looking at the wrong place. I looked at the test build [here](https://gscweb.gsc.wustl.edu/gscmnt/gc9006/info/model_data/2888676250/build9dc0385a1b634c9bb85eb2017b3a0c73/variants/snv/varscan-somatic-2.2.6-d41d8cd98f00b204e9800998ecf8427e/varscan-high-confidence-v1-d41d8cd98f00b204e9800998ecf8427e/) for example, files like `snvs.Germline.hc` and `snvs.LOH.hc` haven't been filtered with the same filters. 
Testing the module on this build, after the filter I'd expect to see about ~234,000 calls in chr1 for this sample, but we see ~280,000 which is closer to the pre-filtered number.